Observing-Interface-Setup/nublado-notebook-environment-update.rst

```diff
@@ -163,7 +167,7 @@ Skip down to the :ref:`summary section <Update-Notebook-Environment-in-Nublado-S
 Here is a summary of the user's command inputs described in the procedure above:
 
 .. code-block:: bash
-   
+
```

Comment:
Because of the way the changes were committed, its hard to actually comment on some things. In the above, step 3 will not work because you aren't in a git directory. After the clone you need to cd first. That's done as part of step 4, but needs to be done earlier.